app/views/layouts/admin.html.erb

```diff
@@ -71,6 +71,14 @@
         </div>
       <% end %>
       <%= yield %>
+      <% if @help_url %>
+        <a class="admin-help-button crayons-btn crayons-btn--icon-rounded crayons-btn--s" href="<%= @help_url %>" target="_blank">
```

Comment:
we should add the usual `rel="noopener noreferer"` so we avoid leaking IPs

Reply:
Cool, added this.